File: 1988-minimize-maximum-pair-sum-in-array/minimize-maximum-pair-sum-in-array.java
class Solution {
    public int minPairSum(int[] nums) {
        Arrays.sort(nums);
        int n = nums.length;
        int ans =0;
        int i=0;
        int j=n-1;
        while(i<j){
            ans =Math.max(ans,nums[i]+nums[j]);
            i++;
            j--;
        }
        return ans;
    }
}

File: 1988-minimize-maximum-pair-sum-in-array/minimize-maximum-pair-sum-in-array.py
class Solution:
    def minPairSum(self, nums: List[int]) -> int:
        nums.sort()
        low = 0
        high = len(nums)-1
        ans = 0
        while low<high:
            ans = max(ans,nums[low]+nums[high])
            low+=1
            high-=1
        return ans

        